src/DesktopClient/PresenceLight/MainWindow.xaml.cs

```diff
@@ -67,6 +67,9 @@ public partial class MainWindow : Window
                           ILogger<MainWindow> logger,
                           ISettingsService settingsService)
         {
+            var currentApp = (App)System.Windows.Application.Current;
```

Comment:
This is the direct answer to your question in Twitter. We grab the "current app" (an existing WPF concept), then grab the service provider that you already created on it.